.circleci/config.yml

```
@@ -32,42 +32,80 @@ jobs:
           path: style-stats.tsv
           destination: style
 
-  test:
-    executor: clojure
+  test-spark-3-1-java-8:
+    executor: clojure-java-8
     steps:
       - checkout
       - restore_cache:
           keys:
-            - v1-test-{{ checksum "project.clj" }}
-            - v1-test-
+            - v1-test-spark-3.1-java-8-{{ checksum "project.clj" }}
+            - v1-test-spark-3.1-java-8-
       - run:
           name: Test projects
-          command: lein monolith each do clean, check, install, test
+          command: |
+            lein -version
```

Prefer the task form of this command:
```suggestion
            lein version
```